src/main/frontend/state.cljs

```diff
@@ -1562,7 +1562,7 @@ Similar to re-frame subscriptions"
 
 (defn active-tldraw-app
   []
-  (when-let [tldraw-el (.closest js/document.activeElement ".logseq-tldraw[data-tlapp]")]
+  (when-let [tldraw-el (.querySelector js/document.body ".logseq-tldraw[data-tlapp]")]
```

Comment:
When tooltips are enabled, the activeElement is modified if the tooltip is activated and `closest` can't find tldraw.